src/workerd/jsg/setup.h

```diff
@@ -414,23 +445,39 @@ class Isolate: public IsolateBase {
       v8::Isolate::CreateParams createParams = {})
       : Isolate(system, nullptr, kj::mv(observer), kj::mv(createParams)) {}
 
+  template <typename MetaConfiguration>
+  void instantiateDefaultWrapper(MetaConfiguration&& configuration) {
+    KJ_DASSERT(wrappers[0].get() == nullptr);
+    auto wrapper = wrapperSpace.construct(ptr, kj::fwd<MetaConfiguration>(configuration));
+    wrapper->initTypeWrapper();
+    wrappers[0] = kj::mv(wrapper);
+  }
+
+  template <typename MetaConfiguration>
+  kj::Own<TypeWrapper>& instantiateWrapper(MetaConfiguration&& configuration) {
```

Comment:
It's a little weird that this returns a reference to a `kj::Own`, especially as it is a reference to a vector slot which could move if the vector is resized. Could you just return `TypeWrapper&` instead? And then `newContextWithWrapper()` would also take `TypeWrapper&` as the param?

Alternatively, maybe the two methods should be combined into a single function `newContextWithSeparateWrapper()` that always allocates a new wrapper for the new context? That avoids leaking a reference to the `TypeWrapper` at all, which seems good since it's meant to be a private implementation type.